spec/spec_helper.rb

```diff
@@ -139,7 +140,18 @@ def restart_driver
     end
   end
 
-  config.before(:all) { restart_driver }
+  # Webmock raises errors that inherit directly from Exception (not StandardError).
+  # The messages contain useful information for debugging stubbed requests to external
+  # services (in tests), but they normally don't appear in the test output.
+  config.before(:all) do
+    ApplicationController.class_eval do
+      rescue_from WebMock::NetConnectNotAllowedError, with: :handle_webmock_error
```

Comment:
there must be something configured on our side (or the absence of it) that hides these errors because I've always seen them without having to hack it like this.

Reply:
I think you are talking about old controller/request specs, not the new feature specs.. this is gffor the new feature specs with the stripe element mocks.

But it's not working very well for me...

Reply:
My point is that this seems rather hackish to me but it's better than nothing :+1:

Reply:
It's really annoying, but the reason is that `WebMock::NetConnectNotAllowedError` inherits directly from `Exception` instead of `StandardError`, so it isn't caught anywhere...

Reply:
I think that's the whole point, tbh